.env

```diff
@@ -1 +1 @@
-PG_URI=''
\ No newline at end of file
+PG_URI=‘postgres://buckbgix:sOue2iVSXg8gJm51HuRZdWRsER-XsSAR@lallah.db.elephantsql.com/buckbgix’
```

Comment:
I believe this was deleted since it's the one in the root.

Reply:
It showed it wasn't in root folder. But anyway, it's deleted.